toolchain/check/testdata/basics/no_prelude/multifile_raw_ir.carbon

```diff
@@ -92,27 +94,29 @@ fn B() {
 // CHECK:STDOUT:   generics:        {}
 // CHECK:STDOUT:   generic_instances: {}
 // CHECK:STDOUT:   types:
-// CHECK:STDOUT:     type0:           {constant: template instNamespaceType, value_rep: {kind: copy, type: type0}}
-// CHECK:STDOUT:     type1:           {constant: template inst+4, value_rep: {kind: none, type: type2}}
-// CHECK:STDOUT:     type2:           {constant: template inst+5, value_rep: {kind: none, type: type2}}
-// CHECK:STDOUT:     type3:           {constant: template inst+10, value_rep: {kind: none, type: type2}}
+// CHECK:STDOUT:     typeTypeType:    {kind: copy, type: typeTypeType}
+// CHECK:STDOUT:     typeError:       {kind: copy, type: typeError}
+// CHECK:STDOUT:     type7:           {kind: copy, type: type7}
```

Comment:
Should this change how type numbers are printed to be offset by the builtin instruction (possibly ConstantId too)? Previously, we had:

```
// CHECK:STDOUT: type1: {constant: template inst+4, value_rep: {kind: none, type: type2}}
```

There, the constant `inst+4` is easy to look up:

```
// CHECK:STDOUT: 'inst+4': {kind: FunctionType, arg0: function0, type: typeTypeType}
```

Now, we have:

```
// CHECK:STDOUT: type7: {kind: copy, type: type7}
```

How do I figure out what `type7` is?

Reply:
Yeah, good point. I've changed printing for `TypeId` to basically print the `ConstantId`, so that you can see which instruction defines the type. For these lines we now produce:
```
// CHECK:STDOUT:     'type(instNamespaceType)': {kind: copy, type: type(instNamespaceType)}
// CHECK:STDOUT:     'type(inst+4)':    {kind: none, type: type(inst+5)}
// CHECK:STDOUT:     'type(inst+5)':    {kind: none, type: type(inst+5)}
// CHECK:STDOUT:     'type(inst+10)':   {kind: none, type: type(inst+5)}
```